how-to/2_Geometry/CreateCustomGeometry.ipynb

```diff
@@ -0,0 +1,410 @@
zj0d69VQHx+dZPZxV6yCP5ScQ4u3Z2Vl/5tkNWRx/aSx1HtZEnp918nU13H80Nhycc0y5oVK/BTAvXry4wbcDUZ5I7jKjvEisy8Fyaq/TSD/Q/bR2pWtn2Y9uBbp8+XI98IEPrKqqhz3sYfX7v//79bjHPa4e9ahH1W/8xm/cYO5mmunWozODTNH84PTZqNtsq/Jzi8mTmfJ3SjgVW4KkpIQlUHd6euV1EK5gyLvHMtJm58CUz2ASnBK0JcWDmyMVOIUDuQJBD4FOGbznnnvWvEjJM0BgoFez+MFGbA+fJ3X5uXKbNu4OsHV9nPrJf7tF3fueaVSfg4HOkOHtSvLrDASeR9+u0Pq47ciVtE4ZZ9sUQqfr7L9OGfT3EyZ5dH1LWbsHnHU56Pe1gnV4e1kfeUx8Mh3TpOvdNbaN9917rWujNcHl4CDNxzDvMRKhmzPpPwG4k1/3+TwCkx3vnq6q1tYshVwKYGpNdJ7dCMZ2nwVYijpPqUfSsG3J4+dz38M1CZzozaRxh+CQ9bBuXafcBNB93rA8f+bUy3CPrA75uXjx4lobSe49FW07hzrAO3p/aWdcmur7EdjvDALk8VajJz7xifWf/tN/qsc85jH1lKc8pV7+8pfXYrGof/yP/3E95jGPudHszXQdqdO7zpuH253ODDLnB6fPTlQc0pHxDHfpLNIdQE2bBjcM9yoSECWAQvJDHqggkDrPpRSIk5OTOjw83Hi/JGUz8tS5rKgIubdVmzYPoNAzmqen9z2rQ9m5MiPFxMGwiCGWSb6kDhwmEEQetqUEGHw80PPKOtzz4MYK/fdQyKQ8J4Dp4CUBOCp8lNXUwuz1sm0OOvwaDSRUbGWwSDwnoOj8EBzzekrTfac6E6CpqjVeU795OzpQPJpz24xHGYc4LykH9VNaX9J/b0cnGzdycS6ndk7NsbSXiQ9+d9Qp5Q4U9ZuhvszroItATs9ndu1wMOqU1v4EDEV+Ci7BWAL1yVNP4Oh9ktbXBFDZPslOvKT+IXj1tU9ezg5oEvD7OkMguLu7W4vFoh1jvral8ZfmdFpnvV9IHirr/ZiiQ7qxw+iE+4OC/r6gr/u6r6t77rmnqqpe8pKX1NOf/vT6xE/8xHq/93u/+tEf/dEbzN1MM916dGaQOT84fXWkxZ7hmV26tKF0CkxS5qmg8VCTpGTwOpVv3wC1ufvJqlMb0cnJycor6B4bfzdhB3wFdlzpodLSndqnNroMHWQpXXpelMDTw4RdyVD/dQCTbUv96HwmuSTqlFSXL5XdBJrSfwc7lOGUxTu1K4F75z3l9/A25qFypvFCzwnnAw9SYfl85Qvrp7KZjD8JIHcAysFoNz5Yb6eocq53a4bzQxmlev1at7543VJSfZ2rWn8sQGlHgHY0DrRuJEMAx2qSf1d3qmMKEPNa8nZ1AJ/vwCSffHbYSTIWMGVaAoOOz1RWN06ZTnLwsFiva5vyEg9sQwJHlA8PRnJjkGTjY5trN9uSgKNOkl0sFqv37PpY8jw8wdzb7mvmaFwS7HLNGslSaWkk6wBn1wcdz+r7Uf6blT7t0z5t9fsxj3lMveUtb6n/9//+Xz3kIQ/ZauzONNNMZ6Mzg8z5wemzU9oIrib0xSkpkaR0EEZS4Ko2lV3fFPW7A1gsxxV6f25Rv7sDdVSOg5IupDGlIwg9PDxcnTK6v7+/8ayO6uczmA7U0nOoV0NTCm5nTJjK70DH+32k4CRjBcv1MaDf7I8RaJlSthwopvyudI4MEgQ8HIvKS+BNr4jz2wE/tjUZXBwcpnJHYLqrM4EJflOOlElX52gMs65Ut4NGPW/YPZ9GUDVaNzqASeWXY9rBnIdwehsSGPRyuv53fjxtKl/pCWb4n6CcXrauXoZvV12Rfwp5dTCaZMux29XrskwyVH6Gx6a0HS9sk4MbB5jqL+ahgYhjheu3+PGQX/XLwcHBxutKXAbKz/cxj2Tj/Uk5cQwxEmAKYDpfI73BPcLd+Nc9Asyr3eNuNnroQx96o1mY6X1A16KnXU8ebnc6M8icH5w+O3mILMN8RK4gePisiEpBClMk8RUhnWKnunwj7ACd52fZrrykQ34YxuRKuvKNrMAEEVLYyB+VBh40pDYdHBxUVa1AhxSeS5cubYBegWS2weXhCl7a8JN8OmW0k7Hue1mpD5yHpOi4zDrFnX1C7yP/T3kydC0p/olX/6aSNFq0XTmjoknFmyBT4Kiq1t6rpz5Nh9qMwKXLbYoSmOLvBMo9LFXtFmjxiIXR3O8UWo4FV4i9zQ4UtJ4QRFVdAfQOLkbgku3svL9cF8Sv5OTy9PYnAO80AvcOGjyfG8c4FgWYuOY46NAYVHkE8twjRvMuRZF4u1Je8uDyY3tSVEBnFKGBT3wkgEw5CGCShymDrK+PDjJpXJIHU+9T9vrTR/uIaAqIJzDZrXFnAZid/DpvJg+fcjmx/m5MzTTTTDOdhc4MMucHp89O3EyrrlhdqzYtjW6ldurAQiJXLKggKz9BG+v0A3WqpkGr+Ds9PV0d8uNKhep3pTIBTZXnyiJl2SkF4lVAkWkODg5WIPj4+HgNYIo/vmrFQ04lJwI03utAP4mKXgISno5ySYDS+5iy28ai523pKCnNXo7/dyOB8jnQSQCLY7STS6pbsqEypTFBYMY6qqoODg425qMU+86o4sol2+k8J3DmgIUfVxaZ1tcVHxsjQ8RIlh2o6pROAnbyxzkiPrU+jF7542Biin/y4eHRCRxPgUrWMerPTi4E2D6WHGDu7OysrVEiAUsHCSybYFNp01q7zdxnvapDZYvHBPQYEeJezM4I4Dx1/S/56eCeqfBNDzXl3OVrWvRhiKwO+knrsc/H/f39tfBcp7Q3+Xe3n5FG49TDY30NcNDJtYLl6j8NauRrpplmmula6Mwgc35w+urJlezrVRaJG2znIXAlLG1mrrBK0dC9zrskgJYOx6FV2ZUNr0v1jJRbbpTcaOlx1PN3lMmFCxfqAQ94QF26dGnjNSXHx8d
```

Comment:
I think this simple geometry example will be really useful for people. I wonder if we should have a separate 2D cone beam, 2D parallel beam, 3D cone beam and 3D parallel beam example?